swagger-ui/dist/zipskee-v1.swagger.yaml

```diff
@@ -55,12 +55,8 @@ paths:
       tags:
        - Auth
       responses:
-        200:
-          description: ok
-        401:
-          description: unauthorized
-          schema:
-            $ref: '#/definitions/Errors'
+        204:
```

Comment:
@michaelsobota When you say "Logout" message - do you want a message that says "User Successfully Logged Out"

Could you elaborate on the redirect?